File: task-3/C/Sub-Task 2.c
#include <stdio.h>

int main() {
    FILE *infile = fopen("input.txt", "r");
    FILE *outfile = fopen("output.txt", "w");
    char data[100];
    fgets(data, 100, infile);
    fputs(data, outfile);
    fclose(infile);
    fclose(outfile);
    return 0;
}

File: task-3/Java/Sub-Task 2.java
import java.nio.file.*;
import java.io.IOException;

public class Subtask2 {
    public static void main(String[] args) throws IOException {
        String data = Files.readString(Paths.get("input.txt"));
        Files.writeString(Paths.get("output.txt"), data);
    }
}
